### PR TITLE
add op-node specific command line flags

### PR DESCRIPTION
**Pull request by roberto-bayardo**
Adds --rollup.sequencerhttp and --rollup.disabletxpoolgossip flags to op-reth node (without implementations) per issue https://github.com/paradigmxyz/reth/issues/3794

### CODE REVIEW

File: bin/reth/src/args/rollup_args.rs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//! clap [Args](clap::Args) for op-reth rollup configuration
+
+/// Parameters for rollup configuration
+#[derive(Debug, clap::Args)]
+#[command(next_help_heading = "Rollup")]
+pub struct RollupArgs {
+    /// HTTP endpoint for the sequencer mempool
+    #[arg(long = "rollup.sequencerhttp", value_name = "HTTP_URL")]
```
Comment (merklefruit) [inline bin/reth/src/args/rollup_args.rs:8]:
```suggestion
    #[arg(long = "rollup.sequencer-http", value_name = "HTTP_URL")]
```

File: bin/reth/src/args/rollup_args.rs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    pub sequencer_http: Option<String>,
+
+    /// Disable transaction pool gossip
+    #[arg(long = "rollup.disabletxpoolgossip")]
```
Comment (merklefruit) [inline bin/reth/src/args/rollup_args.rs:12]:
```suggestion
    #[arg(long = "rollup.disable-tx-pool-gossip")]
```

**Review by merklefruit — APPROVED** [review]
smol nits to keep notation consistent with other arg files, otherwise lgtm

**Review by refcell — APPROVED** [review]
Other than @merklefruit's nits, lgtm

**Comment by roberto-bayardo** [comment]
> Other than @merklefruit's nits, lgtm

I took the flag names verbatim from op-geth. Was torn between consistency with op-geth and consistency with the flag style used in reth!

**Pull request merged**